lambdas/src/contact-retrieve-transcript/index.ts

```diff
@@ -0,0 +1,117 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { S3 } from 'aws-sdk';
+import { SQS } from 'aws-sdk';
+import type { SQSBatchResponse, SQSEvent, SQSRecord, SNSMessage } from 'aws-lambda';
+import { exportTranscript } from './exportTranscript';
+import { getParameters } from './getParameters';
+import { uploadTranscript } from './uploadTranscript';
+
+/**
+ * This is based around latest SQS error handling that supports batchItemFailure responses.
+ *
+ * Reference: https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html#services-sqs-batchfailurereporting
+ */
+
+const sqs = new SQS();
+const CompletedQueueUrl = process.env.completed_sqs_queue_url;
+
+if (!CompletedQueueUrl) {
+  throw new Error('Missing completed_sqs_queue_url ENV Variable');
+}
+
+const processRecord = async (sqsRecord: SQSRecord) => {
+  console.dir(sqsRecord);
+  /**
+   * TODO: This is still based around sending messages with SNS instead of directly to SES. I think we should
+   * consider moving to sending SES directly from HRM, as that will simplify the message structure. If we
+   * were sending a generic "contactSaved" event to SNS and letting sns handle figuring out if it should
+   * process a transcript or a recording based on the payload, the SNS message structure would seem more
+   * worth the complexity. But if we know at job creation time that the job what type of job is being sent
+   * and what jobs should run, we can just send the payload directly to SES and avoid the added SNS message
+   * complexity.
+   */
```

Comment:
While this is true, I think the intent of dropping in SNS was to "have a single outgoing point for pending jobs", rather than having to keep one more variable per job type. Not a big deal, it was just a call to let the infrastructure leverage that knowledge, and not the service itself. I see no harm moving to the "no SNS approach" so happy to discuss with the team.